apps/src/applab/applab.js

```diff
@@ -1332,10 +1335,17 @@ Applab.onCodeModeButton = function() {
 /**
  * If the filename is relative (contains no slashes), then prepend
  * the path to the assets directory for this project to the filename.
+ *
+ * If the filename is external and non-https, route it through the MEDIA_PROXY.
  * @param {string} filename
  * @returns {string}
  */
 Applab.maybeAddAssetPathPrefix = function (filename) {
+
+  if (/^http:\/\//.test(filename)) {
```

Comment:
As far as I can tell, RegExp.new(str) isn't a thing in JS. Presumably you just mean new RegExp(str)?